content/docs/dvclive/dvclive-with-dvc.md

```diff
@@ -7,22 +7,26 @@ ways:
   `dvc metrics` and `dvc plots`. Those same outputs can be visualized in
   [Iterative Studio](#iterative-studio).
 
-- You can monitor model performance in realtime with the
-  [HTML report](#html-report) that DVCLive generates when used alongside DVC.
-
 - DVCLive is also capable of generating [checkpoint](#checkpoints) signal files
   used by DVC <abbr>experiments<abbr>.
 
 ## Setup
 
 We will refer to a training script (`train.py`) already using `dvclive`:
 
+<admon type="tip">
+
+If you use one of the supported [ML Frameworks](/doc/dvclive/ml-frameworks), you
+can jump to its corresponding page to find an example usage.
```

Comment:
Link?
```suggestion
can jump to its [corresponding page](/doc/dvclive/ml-frameworks) to find
example usage.
```

Reply:
That link is already present in the previous line. Do you think it's worth repeating?

Reply:
I guess I didn't notice the link. It's OK then!